code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -194,7 +194,9 @@
 	if(.)
 		amount_grown += rand(1,2)
 		if(amount_grown >= 100)
-			new /mob/living/simple_animal/chicken(src.loc)
+			var/mob/living/simple_animal/chicken/C = new /mob/living/simple_animal/chicken(src.loc)
+			if(key)

```

Comment:
Yeah, this could cause issues with the Staff of Change turning people into chicks, who then grow into chicken and lose their notes, and any objectives they might have had.